src/riak_kv_qry_buffers.erl

```diff
@@ -700,34 +666,29 @@ compute_chunk_size(Data) ->
 
 %% buffer list maintenance
 
-ensure_qref(SQL, QBufs) ->
-    case make_qref(SQL) of
-        {ok, QBufRef} ->
-            case get_qbuf_record(QBufRef, QBufs) of
-                #qbuf{} ->
-                    {ok, {existing, QBufRef}};
-                false ->
-                    {ok, {new, QBufRef}}
-            end;
-        {error, _} = ErrorReason ->
-            ErrorReason
-    end.
+get_qref(_SQL, _QBufs) ->
+    AlwaysUniqueId = crypto:rand_bytes(8),
```

Comment:
`crypto:rand_bytes/1` is deprecated. Maybe we don't care, but if this is going to be a long-term solution I'd suggest `crypto:strong_rand_bytes/1` instead.